packages/sanity/src/ui-components/__workshop__/TooltipStory.tsx

```diff
@@ -0,0 +1,71 @@
+import {Box, Card, Container, Stack, Text} from '@sanity/ui'
+import {useString} from '@sanity/ui-workshop'
+import {Tooltip} from '../tooltip'
+
+export default function TooltipStory() {
+  const text = useString('Tooltip content', 'Tooltip content', 'Props') || ''
+
+  return (
+    <Container width={0} padding={4}>
+      <Stack space={4}>
+        <Card>
+          <Text size={1}>
+            The <code>Tooltip</code> component enforces animation and will correctly style tooltip
+            content when provided as a string (recommended). It also provides affordances for
+            displaying hotkeys.
+          </Text>
+        </Card>
+
+        <Card border radius={2}>
+          <Box marginBottom={3} padding={3}>
+            <Text size={2} weight="medium">
+              Recommended
+            </Text>
+          </Box>
+          <Stack padding={3} space={4}>
+            <Stack space={3}>
+              <Text muted size={1}>
+                String content
+              </Text>
+              <Tooltip content={text}>
+                <Text>Default tooltip with string content</Text>
```

Comment:
nit: Could we adapt this and the other text description to something more generic like, "hover me", or add an icon / button to easy identify where we should hover?

Or change it text to be specific to the tooltip it's representing.

It's a bit confusing to see in all the examples: `Default tooltip with string content`